packages/@glimmer/interfaces/lib/components/component-manager.d.ts

```diff
@@ -80,68 +80,6 @@ export interface WithUpdateHook<ComponentInstanceState = unknown>
   update(state: ComponentInstanceState, dynamicScope: Option<DynamicScope>): void;
 }
 
-// export interface ComponentManager<
```

Comment:
I think this was leftover from an earlier PR, removing as part of general cleanup